esy.json

```diff
@@ -186,6 +190,80 @@
+    "@opam/ppx_custom_printf": "janestreet/ppx_custom_printf:ppx_custom_printf.opam#b809ddfa0b8b97980becd58a2a5929266719cb38",
+    "@opam/ppx_bin_prot": {
+      "version": "opam:v0.14.0",
+      "override": {
+        "dependencies": {
+          "@opam/ppx_sexp_conv": "*",
+          "@opam/bin_prot": "*"
+        }
+      }
+    },
+    "@opam/ppx_sexp_value": {
+      "version": "opam:v0.14.0",
+      "override": {
+        "dependencies": {
+          "@opam/ppx_sexp_conv": "*"
+        }
+      }
+    },
+    "@opam/base_bigstring": {
+      "version": "opam:v0.14.0",
+      "override": {
+        "dependencies": {
+          "@opam/ppx_jane": "*"
+        }
+      }
+    },
+    "@opam/ppx_jane": {
+      "version": "opam:v0.14.0",
+      "override": {
+        "dependencies": {
+          "@opam/ppx_sexp_conv": "*",
+          "@opam/ppx_sexp_message": "*",
+          "@opam/base_quickcheck": "*",
+          "@opam/ppx_custom_printf": "*",
+          "@opam/ppx_expect": "*",
+          "@opam/ppx_inline_test": "*",
+          "@opam/ppx_bench": "*"
+        }
+      }
+    },
+    "@opam/expect_test_helpers_core": "janestreet/expect_test_helpers_core:expect_test_helpers_core.opam#579c7c6f727dd136d79fb0e0c5e19fa6dfd5c409",
+    "@opam/ppx_bench": "janestreet/ppx_bench:ppx_bench.opam#235db81cadf9fec7d48fbac5458f48830407b5b8",
+    "@opam/splittable_random": {
+      "version": "opam:v0.14.0",
+      "override": {
+        "dependencies": {
+          "@opam/ppx_sexp_message": "*",
+          "@opam/ppx_inline_test": "*",
+          "@opam/ppx_bench": "*"
+        }
+      }
+    },
+    "@opam/base_quickcheck": "janestreet/base_quickcheck:base_quickcheck.opam#e22f2748ec48f6c1f4e39acbf78164032023b654",
+    "@opam/ppx_expect": "EduardoRFS/ppx_expect:ppx_expect.opam#5bd917215a86edcff003bcb4d3825620a94215c6",
+    "@opam/sexplib": {
+      "source": "EduardoRFS/sexplib:sexplib.opam#58602935946f7677413a3a367887d0fee281eb26",
+      "override": {
+        "dependencies": {
+          "@opam/bigarray-compat": "*"
+        }
+      }
+    },
+    "@opam/textutils": "janestreet/textutils:textutils.opam#d98ceebe90b86c20611bbcc1741ddedae8015947",
+    "@opam/textutils_kernel": "janestreet/textutils_kernel:textutils_kernel.opam#99e6218f33e51d37bbed9ab620ec8260ad89a66d",
+    "@opam/core_kernel": "janestreet/core_kernel:core_kernel.opam#331c34e99e3f15116f9f0cd6235f1d016adca6d7",
+    "@opam/int_repr": "janestreet/int_repr:int_repr.opam#52e9ba15ecef9d6ae1346b9503c6534bbbab212e",
+    "@opam/core_unix": "EduardoRFS/core_unix:core_unix.opam#e765288323c8d54732e253e34daea6da9b824173",
+    "@opam/timezone": "janestreet/timezone:timezone.opam#e7a86ae5c833de8615265b9d8f598653b5593b91",
+    "@opam/ocaml_intrinsics": "janestreet/ocaml_intrinsics:ocaml_intrinsics.opam#98af7a639c1b4a50c9e4b04e9b070a28129ca6a9",
+    "@opam/sexp_pretty": "EduardoRFS/sexp_pretty:sexp_pretty.opam#50f666f9c464ab8e080f585a0ce72d5b0da43452"
   }
-}
+}
```

Comment:
Can you put back this newline char?

Reply:
Done. For anyone else confused as to why this can matter: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file